### PR TITLE
Complete Iteration 1

### PR DESCRIPTION
**Pull request by Sterling47**
Added functions to display user info and average steps

### CODE REVIEW

**Comment by sanghoro** [comment]
Looks good! thanks!

**Comment by baileyjarvis2814** [comment]
This was completed via liveshare over zoom

**Review by sanghoro — APPROVED** [review]
fantastic!

File: src/domUpdates.js
Comment (baileyjarvis2814) [inline src/domUpdates.js:1]: This is an excellent start

**Pull request merged**